docs/serverless/03-01-function-specification.md

```diff
@@ -130,7 +130,7 @@ See the detailed descriptions of these fields:
 |-------|-------------|
 | **function-name** | Name of the invoked Function |
 | **timeout** | Time, in seconds, after which the system cancels the request to invoke the Function |
-| **runtime** | Environment used to run the Function. You can use `nodejs12`, `nodejs14`, or `python3.8`. |
+| **runtime** | Environment used to run the Function. You can use `nodejs12`, `nodejs14`, `python3.8`, or `python3.9`. |
```

Comment:
I think it should rather be ``` `python38`, or `python39` ``` as [here](https://github.com/kyma-project/kyma/pull/11498/files?file-filters%5B%5D=.md#diff-e31ffa8f1121d0d78b8404c1a4d4340673c1d3fc928ff2b3baae4365123d3667R117), right?

Reply:
Yes, sorry, missed that one!